sdk/videoanalyzer/Azure.Media.VideoAnalyzer.Edge/README.md

```diff
@@ -24,13 +24,13 @@ ServiceClient serviceClient = ServiceClient.CreateFromConnectionString(connectio
 
 ### Install the package
 
-Install the Video Analyzer Edge client library for .NET with NuGet:
+Install the Video Analyzer Edge client library for .NET with [NuGet](https://www.nuget.org/packages/Azure.Media.VideoAnalyzer.Edge):
```

Comment:
This should be to https://nuget.org. See my comment on the other PR for more details, or just close this PR since it seems like a subset of the other PR.